### ISSUE TITLE
question about result reproduction

### ISSUE THREAD

**Issue by KwonGihyun** [lyndonzheng/F-LSeSim#10]
Hi, thanks for your nice work

I am trying to reproduce the result with your pre-trained model,

but the results are not matched to those reported in the paper.

Are the models the same model in your paper? or re-trained version?

If it is same model, please further specify how to get the values in the paper.

Best Regards,

**Comment by Lucky0775** [comment]
> Hi @KwonGihyun, thanks for your interest. The pre-trained model is for best visualization results, rather than the best score. Because the current projects also do the same thing. To do a fair comparison, we follow their reports. Please refer to the duplicate of [taesungp/contrastive-unpaired-translation#65](https://github.com/taesungp/contrastive-unpaired-translation/issues/65) and #1.

Hi,when evaluating the FID of the cityscape dataset, are you use the already processed validation set shared in CUT? Or your preprocessed validation set? Can you share your validation set?
